lighty-modules/lighty-gnmi/lighty-gnmi-commons/src/main/java/io/lighty/modules/gnmi/commons/util/DataConverter.java

```diff
@@ -59,43 +61,44 @@ private DataConverter() {
     public static String jsonStringFromNormalizedNodes(@NonNull final YangInstanceIdentifier identifier,
                                                        @NonNull final NormalizedNode data,
                                                        @NonNull final EffectiveModelContext context) {
-        return toJson(toSchemaPath(identifier), data, context);
+        final JSONCodecFactory jsonCodecFactory = JSONCodecFactorySupplier.RFC7951.createSimple(context);
```

Comment:
IMO: The original solution was more elegant. Now you have to deal with the same problem in createJsonWithNestedWriter and also createJsonWithExclusiveWriter. In addition you deal with it once with isEmpty and once with isPresent :)

I would prefer to just adapt toSchemaPath method.